harc_game_web/apps/tasks/templates/tasks/upload.html

```diff
@@ -95,11 +103,19 @@ <h1 class="title">
       maxChunkSize: 100000, // Chunks of 100 kB
       formData: form_data,
       add: function(e, data) { // Called before starting upload
-        // If this is the second file you're uploading we need to remove the
-        // old upload_id and just keep the csrftoken (which is always first).
-        form_data.splice(1);
-        calculate_md5(data.files[0], 100000);  // Again, chunks of 100 kB
-        data.submit();
+        var uploadErrors = [];
+        if(data.files[0]['size'] > 150000000) {
+          uploadErrors.push('Wybrany plik jest za duży');
```

Comment:
Nie wiem jak to działa bo nie sprawdzałem, ale jak nie, to może wyświetlać błędy przez 'messages' framework

Reply:
ok - następnym razem, to było na szybko robione :D